components/text/dwdiff/dwdiff.license

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
This is copy of the `COPYING` file.  It is better to do not add it to git and instead add `COMPONENT_LICENSE_FILE = COPYING` to `Makefile`.  This will avoid possible future usage of outdated copy of the license.